src/models/DefaultFilters.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const ElectronConfig = require('electron-config')
+const configName = process.env.NODE_ENV === 'test' ? 'config-test' : 'config'
+const storage = new ElectronConfig({ name: configName })
+const Filter = require('./Filter')
+const LastFilter = require('./LastFilter')
+const DEFAULT_FILTER_KEY = 'default-filters'
+
+class DefaultFilters {
+  static init(login) {
+    if (!storage.has(DEFAULT_FILTER_KEY)) {
+      this.myIssuesFilter(login)
+      this.myCommentsFilter(login)
+      this.myAssignmentsFilter(login)
+      this.myMentionsFilter(login)
+      this.myPopularFilter(login)
+      this.save()
+    }
+    return
+  }
+
+  static myIssuesFilter(login) {
+    const defaultFilter = new Filter('My Issues')
+    defaultFilter.store(`author:${login} is:open sort:updated-desc`)
+    LastFilter.save(defaultFilter.key)
+  }
+
+  static myCommentsFilter(login) {
+    const defaultFilter = new Filter('My Comments')
+    defaultFilter.store(`commenter:${login} sort:updated-desc`)
```

Comment:
Since these myXFilter methods are very similar in code, we could maybe DRY things up by using an object.

```javascript
const DEFAULT_FILTERS = {
  'My Issues': 'author:login is:open sort:updated-desc',
  'My Comments': 'commenter:login sort:updated-desc',
  ...
}

static addDefaultFilters() {
  const filterNames = Object.keys(DEFAULT_FILTERS)
  filterNames.forEach(name => {
    const filter = new Filter(name)
    const value = DEFAULT_FILTERS[name].replace('login', login)
    filter.store(value)
  })
  LastFilter.save(filterNames[0])
  storage.set(DEFAULT_FILTER_KEY, 'default-filters')
}
```

Then `init` can just call this method, or do it inline even. Thoughts on this?